.github/workflows/ci.yml

```diff
@@ -31,6 +31,8 @@ jobs:
             django-version: 5.1
           - python-version: 3.9
             django-version: 5.1
+          - python-version: 3.13.0-rc.2
+            django-version: 4.2
```

Comment:
It seems that Django 4.2 does not support 3.13 right now this failure. See: https://github.com/soynatan/django-easy-audit/actions/runs/10855276442/job/30127523880?pr=310

Reply:
Yes, and it won't support it. Django 5.1.3 will be the first version to support Python 3.13 (https://code.djangoproject.com/ticket/34900).